vcsurl_test.go

```diff
@@ -94,14 +94,26 @@ func TestGitLab(t *testing.T) {
 	AssertEqual(t, IsFile(url), true)
 	AssertEqual(t, IsRepo(url), false)
 	AssertEqual(t, IsRawFile(url), false)
-	AssertEqual(t, GetRawFile(url).String(), "https://gitlab.com/gitlab-org/gitlab-services/design.gitlab.com/raw/master/LICENSE")
-	AssertEqual(t, GetRawRoot(url).String(), "https://gitlab.com/gitlab-org/gitlab-services/design.gitlab.com/raw/master/")
+	AssertEqual(t, GetRawFile(url).String(), "https://gitlab.com/gitlab-org/gitlab-services/design.gitlab.com/-/raw/master/LICENSE")
+	AssertEqual(t, GetRawRoot(url).String(), "https://gitlab.com/gitlab-org/gitlab-services/design.gitlab.com/-/raw/master/")
 	AssertEqual(t, IsRawRoot(GetRawRoot(url)), true)
 	AssertEqual(t, GetRepo(url).String(), "https://gitlab.com/gitlab-org/gitlab-services/design.gitlab.com")
 	AssertEqual(t, GetRepo(GetRawRoot(url)).String(), "https://gitlab.com/gitlab-org/gitlab-services/design.gitlab.com")
 
```

Comment:
```suggestion
	// Gitlab backward compatibility
	url, _ = url.Parse("https://riuso.comune.salerno.it/root/simel_2/blob/master/publiccode.yml")
	AssertEqual(t, IsFile(url), true)
	AssertEqual(t, IsRepo(url), false)
	AssertEqual(t, IsRawFile(url), false)
	AssertEqual(t, GetRawFile(url).String(), "https://riuso.comune.salerno.it/root/simel_2/raw/master/publiccode.yml")
	AssertEqual(t, GetRawRoot(url).String(), "https://riuso.comune.salerno.it/root/simel_2/raw/master/")
	AssertEqual(t, IsRawRoot(GetRawRoot(url)), true)
	AssertEqual(t, GetRepo(url).String(), "https://riuso.comune.salerno.it/root/simel_2")
	AssertEqual(t, GetRepo(GetRawRoot(url)).String(), "https://riuso.comune.salerno.it/root/simel_2")
```

I would add these tests to ensure backward compatibility for gitlab older versions

Reply:
I consolidated those in the tests I committed. :+1: